src/components/safe-messages/MsgModal/index.tsx

```diff
@@ -93,6 +97,40 @@ const MsgModal = ({
 
   const isDisabled = !isOwner || hasSigned || !onboard
 
+  const [isPolling, setIsPolling] = useState(false)
+
+  // TODO: move to hook
+  const [ongoingMessage, setOngoingMessage] = useState<Omit<SafeMessage, 'type'>>()
```

Comment:
```suggestion
  const [ongoingMessage, setOngoingMessage] = useState<Omit<SafeMessage, 'type'>>(alreadyProposedMessage)
```

Currently if the message was already proposed we show an error "Your connected wallet has already signed this message."
I think we should then also start polling for that message.